restore/data.go

```diff
@@ -346,7 +361,9 @@ func restoreDataFromTimestamp(fpInfo filepath.FilePathInfo, dataEntries []toc.Co
 func CreateInitialSegmentPipes(oidList []string, c *cluster.Cluster, connectionPool *dbconn.DBConn, fpInfo filepath.FilePathInfo) int {
 	// Create min(connections, tables) segment pipes on each host
 	var maxPipes int
-	if connectionPool.NumConns < len(oidList) {
+	if !backupConfig.SingleDataFile {
```

Comment:
Maybe add a comment here to emphasize this negation (that we are creating a single pipe only if we are *not* in SingleDataFile mode), this condition looks too similar to maxHelpers above

Reply:
added